packages/expo-web-browser/src/ExpoWebBrowser.web.ts

```diff
@@ -105,7 +105,7 @@ export default {
       );
     }
     // Send the URL back to the opening window.
-    parent.postMessage({ url, expoSender: handle }, parent.location);
+    parent.postMessage({ url, expoSender: handle }, `${parent.location}`);
```

Comment:
Slight preference to use parent.location.toString() here so we don't silently specify `"undefined"` if parent.location is undefined

Reply:
In what cases [`window.location`](https://developer.mozilla.org/en-US/docs/Web/API/Window/location) is `undefined`? 🤔 I don't see it might be `null` or `undefined` in the the typings as well: https://github.com/microsoft/TypeScript/blob/88d8d1cceeb0fa83d849f9b9f374c59e9a33b7a4/lib/lib.dom.d.ts#L17193 🤔

Reply:
If it's never undefined, then location.toString() should always work. If for some reason it is undefined, then we throw an error (good).